cmd/kpod/common.go

```diff
@@ -133,3 +133,320 @@ func validateFlags(c *cli.Context, flags []cli.Flag) error {
+		Usage: "Limit the CPU CFS (Completely Fair Scheduler) quota",
+	},
+	cli.Uint64Flag{
+		Name:  "cpu-rt-period",
+		Usage: "Limit the CPU real-time period in microseconds",
+	},
+	cli.Int64Flag{
+		Name:  "cpu-rt-runtime",
+		Usage: "Limit the CPU real-time runtime in microseconds",
+	},
+	cli.Uint64Flag{
+		Name:  "cpu-shares",
+		Usage: "CPU shares (relative weight)",
+	},
+	cli.StringFlag{
+		Name:  "cpus",
+		Usage: "Number of CPUs. The default is 0.000 which means no limit",
+	},
+	cli.StringFlag{
+		Name:  "cpuset-cpus",
+		Usage: "CPUs in which to allow execution (0-3, 0,1)",
+	},
+	cli.StringFlag{
+		Name:  "cpuset-mems",
+		Usage: "Memory nodes (MEMs) in which to allow execution (0-3, 0,1). Only effective on NUMA systems.",
+	},
+	cli.BoolFlag{
+		Name:  "detach, d",
+		Usage: "Run container in background and print container ID",
+	},
+	cli.StringFlag{ //
+		Name:  "detach-keys",
+		Usage: "Override the key sequence for detaching a container. Format is a single character `[a-Z]` or `ctrl-<value>` where `<value>` is one of: `a-z`, `@`, `^`, `[`, `,` or `_`",
+	},
+	cli.StringSliceFlag{
+		Name:  "device",
+		Usage: "Add a host device to the container (default [])",
+	},
+	cli.StringSliceFlag{
+		Name:  "device-read-bps",
+		Usage: "Limit read rate (bytes per second) from a device (e.g. --device-read-bps=/dev/sda:1mb)",
+	},
+	cli.StringSliceFlag{
+		Name:  "device-read-iops",
+		Usage: "Limit read rate (IO per second) from a device (e.g. --device-read-iops=/dev/sda:1000)",
+	},
+	cli.StringSliceFlag{
+		Name:  "device-write-bps",
+		Usage: "Limit write rate (bytes per second) to a device (e.g. --device-write-bps=/dev/sda:1mb)",
+	},
+	cli.StringSliceFlag{
+		Name:  "device-write-iops",
+		Usage: "Limit write rate (IO per second) to a device (e.g. --device-write-iops=/dev/sda:1000)",
+	},
+	cli.StringSliceFlag{
+		Name:  "dns",
+		Usage: "Set custom DNS servers",
+	},
+	cli.StringSliceFlag{
+		Name:  "dns-opt",
+		Usage: "Set custom DNS options",
+	},
+	cli.StringSliceFlag{
+		Name:  "dns-search",
+		Usage: "Set custom DNS search domains",
+	},
+	cli.StringFlag{
+		Name:  "entrypoint",
+		Usage: "Overwrite the default ENTRYPOINT of the image",
+	},
+	cli.StringSliceFlag{
+		Name:  "env, e",
+		Usage: "Set environment variables in container",
+	},
+	cli.StringSliceFlag{
+		Name:  "env-file", //
+		Usage: "Read in a file of environment variables",
+	},
+	cli.StringSliceFlag{
+		Name:  "expose",
+		Usage: "Expose a port or a range of ports (default [])",
+	},
+	cli.StringSliceFlag{
+		Name:  "group-add",
+		Usage: "Add additional groups to join (default [])",
+	},
+	cli.StringFlag{
+		Name:  "hostname",
+		Usage: "Set container hostname",
+	},
+	cli.BoolFlag{
+		Name:  "interactive, i",
+		Usage: "Keep STDIN open even if not attached",
+	},
+	cli.StringFlag{
+		Name:  "ip",
+		Usage: "Container IPv4 address (e.g. 172.23.0.9)",
+	},
+	cli.StringFlag{
+		Name:  "ip6",
+		Usage: "Container IPv6 address (e.g. 2001:db8::1b99)",
```

Comment:
suggest 2001:0db8:AC10:1b99 for the address.  I'm not sure the 3 char is ok nor the blank.

Reply:
Actually, his v6 address is more correct than yours. Leading 0s are omitted, all-0 blocks may be omitted at most once with a double-colon to indicate one or more has been elided. Yours is too short by a fair bit

Reply:
If that's valid, I'm fine with leaving.  FWIW, when I made my comment I looked at the Wikipedia for IPV6 and they'd listed "2001:0db8:0000:0042:0000:8a2e:0370:7334", which I modified to keep the same size as the example.

Reply:
Both forms are correct (minus yours being shortened), but the one @baude used is the one in common use - v6 addresses are too long already, and typing all the extra 0s would make things worse.